pipeline_tools/http_requests.py

```diff
@@ -159,7 +159,7 @@ def is_retryable(error):
             print('{0} {1}'.format(now, repr(error)))
 
             def is_retryable_status_code(error):
-                return isinstance(error, requests.HTTPError) and not (400 <= error.response.status_code <= 499)
+                return isinstance(error, requests.HTTPError) and not (400 <= error.response.status_code <= 499 and error.response.status_code != 409)
```

Comment:
Hmm, this condition is getting a bit long and confusing.

What about something like:
```
if not isinstance(error, requests.HTTPError):
    return False
if error.response.status_code == 409:
    return True
else:
    return not (400 <= error.response.status_code <= 499)
```